src/data/mystery-encounters/mystery-encounter-utils.ts

```diff
@@ -294,7 +349,7 @@ export async function initBattleWithEnemyConfig(scene: BattleScene, partyConfig:
   } else {
     // Wild
     scene.currentBattle.mysteryEncounter.encounterVariant = MysteryEncounterVariant.WILD_BATTLE;
-    battle.enemyLevels = new Array(partyConfig?.pokemonConfigs?.length > 0 ? partyConfig?.pokemonConfigs?.length : doubleBattle ? 2 : 1).fill(null).map(() => scene.currentBattle.getLevelForWave());
+    battle.enemyLevels = new Array(normalCount + bossCount > 0 ? normalCount + bossCount : partyConfig?.doubleBattle ? 2 : 1).fill(null).map(() => scene.currentBattle.getLevelForWave());
```

Comment:
I think something might have been messed up in this function when merging, looks like this function is being reverted to what it was a few commits ago. Unless there’s a specific reason, all these changes in this function can be removed I think.

Reply:
good catch, i mustve messed that up